tests/test_utils/test_assigner.py

```diff
@@ -628,3 +628,15 @@ def test_mask_hungarian_match_assigner():
         dice_cost=dict(type='DiceCost', weight=0.0, pred_act=True, eps=1.0))
     with pytest.raises(AssertionError):
         self = MaskHungarianAssigner(**assigner_cfg)
+
+    # test with dicecost used in K-Net
+    assigner_cfg = dict(
+        cls_cost=dict(type='ClassificationCost', weight=0.0),
+        dice_cost=dict(
+            type='DiceCost',
+            weight=0.0,
+            pred_act=True,
+            eps=1.0,
+            naive_dice=False))
+    with pytest.raises(AssertionError):
```

Comment:
We should not only test AssertionError, we should also test the case when it can run normally.